packages/components/src/panel/body.tsx

```diff
@@ -93,7 +101,15 @@ export function PanelBody(
 }
 
 const PanelBodyTitle = forwardRef(
-	( { isOpened, icon, title, ...props }, ref ) => {
+	(
+		{
+			isOpened,
+			icon,
+			title,
+			buttonProps,
```

Comment:
Since we previously used to pass all other props down, `buttonProps` is going to be missing `onClick`.

Should be easy to fix if we manually extract it from here and pass it to  `<Button />` below manually.

This is one of the reasons why the tests are currently failing.

Reply:
Ah, that makes sense. Thanks for sharing!